### PR TITLE
Update Helm Broker component version, documentation and parameter

### PR DESCRIPTION
**Pull request by jasiu001**
Change version of Helm Broker component, documentation and parameter name. Related to #1255

### CODE REVIEW

File: docs/service-brokers/docs/011-configuration-helm-broker.md [hunk 1]
```diff
@@ -3,14 +3,14 @@ title: Configure Helm Broker
...
-The Helm Broker fetches bundle definitions from an HTTP server defined in the `values.yaml` file. The **config.repository.baseURL** attribute defines the HTTP server URL.
+The Helm Broker fetches bundle definitions from an HTTP server defined in the `values.yaml` file. The **config.repository.URL** attribute defines the HTTP server URL.
 
 ### Configuring the Helm Broker externally
 
 Follow these steps to change the configuration and make the Helm Broker fetch bundles from a custom HTTP server:
 
 1. Create a remote bundles repository. Your remote bundle repository must include the following resources:
-    - An `index.yaml` file which defines available bundles.
+    - An `yaml` file which defines available bundles, for example `bundles.yaml`.
```
Comment (klaudiagrz) [inline docs/service-brokers/docs/011-configuration-helm-broker.md]: ``A `yaml` file``

File: docs/service-brokers/docs/011-configuration-helm-broker.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ Follow these steps to change the configuration and make the Helm Broker fetch bu
             description: {bundle_description}
             version: {bundle_version}
       ```
-      This is an example of an `index.yaml` file for the Redis bundle:
+      This is an example of an `yaml` file for the Redis bundle:
```
Comment (klaudiagrz) [inline docs/service-brokers/docs/011-configuration-helm-broker.md]: ``a `yaml` file ``

File: docs/service-brokers/docs/011-configuration-helm-broker.md [hunk 3]
```diff
@@ -31,14 +31,15 @@ Follow these steps to change the configuration and make the Helm Broker fetch bu
...
 
-2. In the `values.yaml` provide your server's URL in the **repository.baseURL** attribute:
+2. In the `values.yaml` provide your server's URL in the **repository.URL** attribute:
 
   ```yaml
     repository:
-      baseURL: "http://custom.bundles-repository"
+      URL: "http://custom.bundles-repository/bundles.yaml"
   ```
+  > **NOTE:** The default filename is `index.yaml`. You can skip `yaml` file name in url if his name is `index.yaml`. In that case, do not provide the filename in the URL. Your **URL** should be equal to `http://custom.bundles-repository/`.
```
Comment (klaudiagrz) [inline docs/service-brokers/docs/011-configuration-helm-broker.md]: ~You can skip `yaml` file name in url if his name is `index.yaml`.~ -> I'd remove this sentence (it repeats what is said in the next one).

Comment (jasiu001) [reply docs/service-brokers/docs/011-configuration-helm-broker.md]: For me if I remove this sentence all has no sense. The third sentence follows from the second.

**Pull request merged**